### DISCUSSION TITLE
Capitalization on addon names

### DISCUSSION THREAD

**Discussion by Hans5958** [ScratchAddons/ScratchAddons#180] [Help]
On the addon names, should we do sentence case (Message count on extension icon) or capitalised case (Colored Context Menus)?

**Comment by NoobTracker** [comment]
I Think That This Looks Kinda Stupid so no I guess.
But currently it's really a mix.

**Comment by WorldLanguages** [comment]
Yeah, before the release we'll have to standardize the naming. Meanwhile, it's fine.

**Comment by Hans5958** [comment]
> 
> 
> Yeah, before the release we'll have to standardize the naming. Meanwhile, it's fine.

We should put it in a milestone, if you know that there are milestones on GitHub, I guess.
